migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+Check the [what-to-migrate.md]() file first, please.
+
+### How we want to do it
+
+We will use the XXX tool (not released yet).
+  - still thinking about its name :D I think about two possibilities:
+    - `volagit` or `forgit` -> these are just random names containing `git`... I think
+       about them because they sounds cool :D
+
+### What tools it will use
+
+- [Ogr](https://github.com/packit/ogr)
+  - A tool which wraps Pagure, Gitlab and Github API into one API
+  - There's 95% of API calls we need to use 
+- [GitPython](https://gitpython.readthedocs.io/en/stable/)
+  - A tool which wraps git cli to a Python api
+  - only when Github, Gitlab API doesn't allow something (e.g. `git reset`). In other cases Ogr
+    includes all the stuff we need.
+- That's it
+  
+### What we need to use to achieve the goals in [what-to-migrate.md]()
+
+- Create a bot account which will transfer the PRs/Issues to Github
```

Comment:
We already have a bot account, but I don't think we need that in particular.  I wouldn't mind if you created those issues under your GH ID, just to spent minimal time on it and to get the job done.

Reply:
I think that should be enough. But if not - I don't think it's a big deal to create new bot account just for this migration. It should take few minutes.